forks/Microsoft.Health.FellowOakDicom/Serialization/JsonDicomConverter.cs

```diff
@@ -41,9 +42,12 @@ public DicomArrayJsonConverter()
         /// Initialize the JsonDicomConverter.
         /// </summary>
         /// <param name="writeTagsAsKeywords">Whether to write the json keys as DICOM keywords instead of tags. This makes the json non-compliant to DICOM JSON.</param>
-        public DicomArrayJsonConverter(bool writeTagsAsKeywords)
+        /// <param name="dropDataWhenInvalid">Whether to drop attribute from JSON when an invalid Dicom JSON attribute is found.
+        ///     Default "false" will produce errors instead of dropping. Does not handle invalid JSON, just invalid Dicom JSON. </param>
+        public DicomArrayJsonConverter(bool writeTagsAsKeywords, bool dropDataWhenInvalid=false)
         {
             _writeTagsAsKeywords = writeTagsAsKeywords;
+            _dropDataWhenInvalid = dropDataWhenInvalid;
```

Comment:
Will we make this change in the fo-dicom library? If not we will need to drop the DicomElements before getting json serialized.